domain/schema/model/sql/0025-model-status.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE model_status_value (
+    id INT PRIMARY KEY,
+    status TEXT NOT NULL
+);
+
+-- Status values for the model.
+INSERT INTO model_status_value VALUES
+(0, 'error'),
+(1, 'available'),
+-- We set the model status to busy when the model is being upgrading.
+(2, 'busy'),
+-- We set the model status to suspended when the model has invlid cloud credentials.
```

Comment:
s/invlid/invalid

I am not sure we need this comment here as we are not going to be setting invalid cloud credentials as a status.